ProcessLib/HydroMechanics/HydroMechanicsProcessData.h

```diff
@@ -82,10 +72,14 @@ struct HydroMechanicsProcessData
     //! Assignments are not needed.
     void operator=(HydroMechanicsProcessData&&) = delete;
 
+    MeshLib::PropertyVector<int> const* material_ids;
```

Comment:
Can the type be reference?

Reply:
No, not with current implementation. It is allowed not to have the 'material_ids'.